### DISCUSSION TITLE
clickhouse as storage

### DISCUSSION THREAD

**Discussion by Cluas** [grafana/phlare#420] [Help and support] [answered]
Why don't we just use clickhouse as storage?

**Comment by cyriltovena** [comment, marked as answer]
Because we run this at scale to power our SaaS Grafana Cloud and we're used to our own architecture for operations, we have no experience with ClickHouse.

I also believe a domain specific storage will always end up being less expensive at scale :), so less expensive for our OSS users and paying customers.
